<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>734</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_num_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_sum">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_min">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_mul">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_div">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_exe">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>51</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_del">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sign_dot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_pow">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sign_opb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sign_clb">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_mod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_sin">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_cos">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_tg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>tg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_acos">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_asin">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_atan">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_ln">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_log">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_op_sqrt">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>471</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="graph_min_x">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="graph_max_x">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_graph_build">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>build</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_graph_x">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_var_x">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="expression_x_replace">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
